torch/csrc/autograd/saved_variable.cpp

```diff
@@ -41,6 +41,21 @@ SavedVariable::SavedVariable(const Variable& variable, bool is_output, bool is_i
     version_counter_ = impl::version_counter(variable);
```

Comment:
Why do you still save the version counter here?

Reply:
Yes I think you're right that I don't need it here.